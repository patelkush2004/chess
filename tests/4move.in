game human human
done
move e7 e5
move e2 e4
move h7 h6
move f1 b5
move a7 a6
move d1 g4
move a6 a5
move g4 d7
