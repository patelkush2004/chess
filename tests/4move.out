   ________
8 |rnbqkbnr|
7 |pppppppp|
6 |_ _ _ _ |
5 | _ _ _ _|
4 |_ _ _ _ |
3 | _ _ _ _|
2 |PPPPPPPP|
1 |RNBQKBNR|

   abcdefgh
White Turn
   ________
8 |rnbqkbnr|
7 |pppppppp|
6 |_ _ _ _ |
5 | _ _ _ _|
4 |_ _ P _ |
3 | _ _ _ _|
2 |PPPP_PPP|
1 |RNBQKBNR|

   abcdefgh
Black Turn
   ________
8 |rnbqkbnr|
7 |pppp ppp|
6 |_ _ _ _ |
5 | _ _p_ _|
4 |_ _ P _ |
3 | _ _ _ _|
2 |PPPP_PPP|
1 |RNBQKBNR|

   abcdefgh
White Turn
   ________
8 |rnbqkbnr|
7 |pppp ppp|
6 |_ _ _ _ |
5 | B _p_ _|
4 |_ _ P _ |
3 | _ _ _ _|
2 |PPPP_PPP|
1 |RNBQK_NR|

   abcdefgh
Black Turn
   ________
8 |rnbqkbnr|
7 | ppp ppp|
6 |_ _ _ _ |
5 |pB _p_ _|
4 |_ _ P _ |
3 | _ _ _ _|
2 |PPPP_PPP|
1 |RNBQK_NR|

   abcdefgh
White Turn
   ________
8 |rnbqkbnr|
7 | ppp ppp|
6 |_ _ _ _ |
5 |pB _p_ Q|
4 |_ _ P _ |
3 | _ _ _ _|
2 |PPPP_PPP|
1 |RNB_K_NR|

   abcdefgh
Black Turn
   ________
8 |rnbqkbnr|
7 | ppp ppp|
6 |_ _ _ _ |
5 | B _p_ Q|
4 |p _ P _ |
3 | _ _ _ _|
2 |PPPP_PPP|
1 |RNB_K_NR|

   abcdefgh
White Turn
   ________
8 |rnbqkbnr|
7 | ppp Qpp|
6 |_ _ _ _ |
5 | B _p_ _|
4 |p _ P _ |
3 | _ _ _ _|
2 |PPPP_PPP|
1 |RNB_K_NR|

   abcdefgh
Black Turn
Black is in check.
